src/kagglehub/auth.py

```diff
@@ -9,20 +12,96 @@
 
 INVALID_CREDENTIALS_ERROR = 403
 
+NOTEBOOK_LOGIN_TOKEN_HTML_START = """<center> <img
+src=https://www.kaggle.com/static/images/site-logo.png
+alt='Kaggle'> <br> Copy a token from <a
+href="https://www.kaggle.com/settings/account" target="_blank">your Kaggle
+settings page</a> and paste it below. <br> </center>"""
 
-def login(validate_credentials=True):  # noqa: FBT002
-    """Prompt the user for their Kaggle username and API key and save them globally."""
+NOTEBOOK_LOGIN_TOKEN_HTML_END = """
+<b>Thank You</b></center>"""
 
-    username = input("Enter your Kaggle username: ")
-    api_key = input("Enter your Kaggle API key: ")
 
-    set_kaggle_credentials(username=username, api_key=api_key)
+@contextmanager
+def capture_logger_output():
+    """Capture output that is logged using the logger.
+
+    Example:
+    ```py
+    >>> output = capture_logger_output()
+    >>> logger.info("This is an info message")
+    >>> logger.error("This is an error message")
+    >>> print(output)
+    ```
+    """
+    buffer = io.StringIO()
+    handler = logging.StreamHandler(buffer)
+    logger = logging.getLogger()
+    logger.addHandler(handler)
+    try:
+        yield buffer
+    finally:
+        logger.removeHandler(handler)
 
-    logger.info("Kaggle credentials set.")
 
-    if not validate_credentials:
-        return
+def is_in_notebook():
```

Comment:
rename to `_is_in_notebook()`. 

The leading `_` is Python's convention for a private method.